### PR TITLE
update contributing and readme to template

### PR DESCRIPTION
**Pull request by elizoller**
**GitHub Issue**: (https://github.com/Islandora-CLAW/CLAW/issues/1267)

[Tech call minutes](https://github.com/Islandora-CLAW/CLAW/wiki/September-25%2C-2019#minutes)

# What does this Pull Request do?
Updates Readme to standards
Updates contributing.md to standards

# What's new?
An updated readme and contributing page

# How should this be tested?

Give it a read and compare it to the [template](https://github.com/Islandora-CLAW/CLAW/blob/master/docs/contributing/readme_template.md).

# Interested parties
@Islandora-CLAW/committers @manez

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
 
 ## Workflows
 
 The group meets each Wednesday at 1:00 PM Eastern. Meeting notes and announcements are posted to the [Islandora community list](https://groups.google.com/forum/#!forum/islandora) and the [Islandora developers list](https://groups.google.com/forum/#!forum/islandora-dev). You can view meeting agendas, notes, and call-in information [here](https://github.com/Islandora-CLAW/CLAW/wiki#islandora-claw-tech-calls). Anybody is welcome to join the calls, and add items to the agenda.
 
 ### Use cases
 
-If you would like to submit a use case to the Islandora CLAW project, please submit an issue [here](https://github.com/Islandora-CLAW/CLAW/issues/new) using the [Use Case template](https://github.com/Islandora-CLAW/CLAW/wiki/Use-Case-template), prepending "Use Case:" to the title of the issue.
+If you would like to submit a use case to the Islandora 8 project, please submit an issue [here](https://github.com/Islandora-CLAW/CLAW/issues/new) using the [Use Case template](https://github.com/Islandora-CLAW/CLAW/wiki/Use-Case-template), prepending "Use Case:" to the title of the issue.
```
Comment (manez) [inline CONTRIBUTING.md]: The great de-CLAWing requires some links to be updated. This is now https://github.com/Islandora/documentation/issues/new

Comment (manez) [reply CONTRIBUTING.md]: And here: https://github.com/Islandora/documentation/wiki/Use-Case-template

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
 
 ### Documentation
 
 You can contribute documentation in two different ways. One way is to create an issue [here](https://github.com/Islandora-CLAW/CLAW/issues/new), prepending "Documentation:" to the title of the issue. Another way is by pull request, which is the same process as [Contribute Code](https://github.com/Islandora-CLAW/CLAW/blob/master/CONTRIBUTING.md#contribute-code). All documentation resides in [`docs`](https://github.com/Islandora-CLAW/CLAW/tree/master/docs).
```
Comment (manez) [inline CONTRIBUTING.md]: https://github.com/Islandora/documentation/issues/new and https://github.com/Islandora/Crayfish-Commons/blob/master/CONTRIBUTING.md#contribute-code and https://github.com/Islandora/documentation/tree/master/docs

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
 
 ### Request a new feature
 
-To request a new feature you should [open an issue in the CLAW repository](https://github.com/Islandora-CLAW/CLAW/issues/new) or create a use case (see the _Use cases_ section above), and summarize the desired functionality. Prepend "Enhancement:" if creating an issue on the project repo, and "Use Case:" if creating a use case.
+To request a new feature you should [open an issue in the Islandora 8 repository](https://github.com/Islandora-CLAW/CLAW/issues/new) or create a use case (see the _Use cases_ section above), and summarize the desired functionality. Prepend "Enhancement:" if creating an issue on the project repo, and "Use Case:" if creating a use case.
```
Comment (manez) [inline CONTRIBUTING.md]: https://github.com/Islandora/documentation/issues/new

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
 
 ### Report a bug
 
-To report a bug you should [open an issue in the CLAW repository](https://github.com/Islandora-CLAW/CLAW/issues/new) that summarizes the bug. Prepend the label "Bug:" to the title of the issue.
+To report a bug you should [open an issue in the Islandora 8 repository](https://github.com/Islandora-CLAW/CLAW/issues/new) that summarizes the bug. Prepend the label "Bug:" to the title of the issue.
```
Comment (manez) [inline CONTRIBUTING.md]: https://github.com/Islandora/documentation/issues/new

File: README.md [hunk 5]
```diff
@@ -1,32 +1,56 @@
...
 
-If you would like to contribute, please get involved by attending our weekly 
-[Tech Call][5]. We love to hear from you!
+## Documentation
 
-If you would like to contribute code to the project, you need to be covered by 
-an Islandora Foundation [Contributor License Agreement][6] or 
-[Corporate Contributor License Agreement][7]. Please see the 
-[Contributors][8] pages on Islandora.ca for more information.
+Further documentation for this module is available on the [Islandora 8 documentation site](https://islandora-claw.github.io/CLAW/).
```
Comment (manez) [inline README.md]: https://github.com/Islandora/documentation/tree/master/docs

File: README.md [hunk 6]
```diff
@@ -39,17 +63,16 @@ an Islandora Foundation [Contributor License Agreement][6] or
 * American Philosophical Society
 * common media inc.
 
-## Maintainers
+## Development
 
-* [Daniel Lamb](https://github.com/dannylamb)
+If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
```
Comment (manez) [inline README.md]: https://github.com/Islandora/documentation/wiki

File: README.md [hunk 7]
```diff
@@ -39,17 +63,16 @@ an Islandora Foundation [Contributor License Agreement][6] or
...
 
-## License
+If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
+
+We recommend using the [claw-playbook](https://github.com/Islandora-Devops/claw-playbook) to get started. If you want to pull down the submodules for development, don't forget to run `git submodule update --init --recursive` after cloning.
```
Comment (manez) [inline README.md]: This should be the last lingering CLAW url

Comment (elizoller) [reply README.md]: this should be fixed now

File: README.md (hunk 6 above)
Comment (elizoller) [reply README.md]: this should be fixed now

File: README.md (hunk 5 above)
Comment (elizoller) [reply README.md]: this should be fixed now

File: CONTRIBUTING.md (hunk 4 above)
Comment (elizoller) [reply CONTRIBUTING.md]: this should be fixed now

File: CONTRIBUTING.md (hunk 3 above)
Comment (elizoller) [reply CONTRIBUTING.md]: this should be fixed now

File: CONTRIBUTING.md (hunk 2 above)
Comment (elizoller) [reply CONTRIBUTING.md]: this should be fixed now

File: CONTRIBUTING.md (hunk 1 above)
Comment (elizoller) [reply CONTRIBUTING.md]: i think these have been resolved

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,26 +1,30 @@
 # Welcome!
 
-If you are reading this document then you are interested in contributing to the Islandora CLAW. All contributions are welcome: use-cases, documentation, code, patches, bug reports, feature requests, etc. You do not need to be a programmer to speak up!
+If you are reading this document then you are interested in contributing to Islandora 8. All contributions are welcome: use-cases, documentation, code, patches, bug reports, feature requests, etc. You do not need to be a programmer to speak up!
+
+We also have an irc channel -- #islandora -- on freenode.net. Feel free to hang out there, ask questions, and help others out if you can.
```
Comment (whikloj) [inline CONTRIBUTING.md]: IRC channel is essentially dead, this line should probably be removed

Comment (elizoller) [reply CONTRIBUTING.md]: should it be replaced with slack or something?

Comment (manez) [reply CONTRIBUTING.md]: Not yet. We're working on auto-invite solutions, but we don't know what that link will look lie yet, so there's no standard text to include.

**Comment by dannylamb** [comment]
For the good of the CLAWdit, I'm citing executive privilege and merging this even though I did push a commit to get Travis going.

**Pull request merged**